contracts/vaults/RibbonThetaVault.sol

```diff
@@ -536,26 +533,57 @@ contract RibbonThetaVault is DSMath, OptionsVaultStorage {
         address newOption = nextOption;
         require(newOption > PLACEHOLDER_ADDR, "!nextOption");
```

Comment:
so newOption > 1 is same as newOption != address(0)?

Reply:
Yep!